api_calls/reference/bookings/create_with_comments_fees_and_taxes-request.json

```diff
@@ -15,13 +15,13 @@
         { "content": "Comment two" }
       ],
       "bookings_fees": [
-        { "price": "50", "times_booked": "2", "rentals_fee_id": "10" },
-        { "price": "30", "times_booked": "1", "rentals_fee_id": "10" }
+        { "price": "50", "times_booked": "2", "rentals_fee_id": "10", "name": "Fee1" },
+        { "price": "30", "times_booked": "1", "rentals_fee_id": "10", "name": "Fee2" }
```

Comment:
nah with `name_en` it throws `bookings_fees.name","title":"can't be blank"`

returns
``` 
"name": {
  "en": "fee"
}
```
though